rules/waf.yaml

```diff
@@ -32,8 +32,17 @@ spec:
       annotations:
         summary: WAF block ratio is greater than 20%
         description: WAF block ratio is {{$value}}% for WAF ACL {{ $labels.dimension_WebACL }} in customer {{ $labels.overwrite_asy_customer }} account {{ $labels.overwrite_aws_account_id }} region {{ $labels.overwrite_aws_region }}
-        runbook_url: https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-troubleshooting.html
+        runbook_url: https://repost.aws/knowledge-center/waf-mitigate-ddos-attacks
       expr: round(rate(aws_cloudwatch_WAFV2_BlockedRequests_Region_Rule_WebACL_sum{dimension_Rule="ALL"}[15m]) / (rate(aws_cloudwatch_WAFV2_AllowedRequests_Region_Rule_WebACL_sum{dimension_Rule="ALL"}[15m]) + rate(aws_cloudwatch_WAFV2_CountedRequests_Region_Rule_WebACL_sum{dimension_Rule="ALL"}[15m]) + rate(aws_cloudwatch_WAFV2_BlockedRequests_Region_Rule_WebACL_sum{dimension_Rule="ALL"}[15m])) * 100) > 20
       for: 30m
       labels:
         severity: warning
+    - alert: AwsCloudwatchWafV2BlockedAttacks
+      annotations:
+        summary: WAF blocked attacks
+        description: WAF blocked {{ $labels.dimension_Attack }} attack for WAF ACL {{ $labels.dimension_WebACL }} in customer {{ $labels.overwrite_asy_customer }} account {{ $labels.overwrite_aws_account_id }} region {{ $labels.overwrite_aws_region }}
+        runbook_url: https://repost.aws/knowledge-center/waf-mitigate-ddos-attacks
+      expr: aws_cloudwatch_WAFV2_BlockedRequests_Attack_WebACL > 10
```

Comment:
Dito, is this added in the whitelist. Either way, it would require a stream stack update across all envs and regions